src/CompoundJoin.sol

```diff
@@ -0,0 +1,318 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Do we need to also support cage()?
I see it in all the adpapters and in sushiJoin.
Can we add it to the CropJoin base contract?

Reply:
Yeah this is probably a good idea for consistency.